install-env.sh

```diff
@@ -11,10 +11,9 @@ cp certs/mainflux-server.key /etc/ssl/private/
 openssl dhparam -out /etc/ssl/certs/dhparam.pem 2048
 
 # Copy nginx config snippets:
+mkdir -p /etc/nginx/snippets
 cp snippets/* /etc/nginx/snippets/
 
-# Copy `sites-avalable/mainflux-proxy`:
-cp sites-available/mainflux-proxy /etc/nginx/sites-available/mainflux-proxy
-
-# Enable it:
-ln -s /etc/nginx/sites-available/mainflux-proxy /etc/nginx/sites-enabled/mainflux-proxy
+# Enable proxy as default:
+mv /etc/nginx/conf.d/default.conf /etc/nginx/conf.d/default.conf.bak
+cp conf.d/mainflux-proxy.conf /etc/nginx/conf.d/mainflux-proxy.conf
```

Comment:
I do agree that it is simpler, and if NGINX dropped it I totally agree that we introduce the changes you have made.

I am running Debian Testing for my personal dev, but Debian is always a bit behind, so maybe my NGINX is a bit rusty.